cmd/soroban-rpc/internal/db/transaction.go

```diff
@@ -223,6 +224,7 @@ func ParseTransaction(lcm xdr.LedgerCloseMeta, ingestTx ingest.LedgerTransaction
 		Sequence:  lcm.LedgerSequence(),
 		CloseTime: lcm.LedgerCloseTime(),
 	}
+	tx.TransactionHash = ingestTx.Result.TransactionHash.HexString()
```

Comment:
I think this is fine here, but I just want to call out that this will give the fee-bump hash rather than the inner transaction hash in the fee-bump case. If people want the inner result, they can open a feature request :+1: